external-crates/move/crates/move-compiler/tests/linter/incorrect_constant_naming.move

```diff
@@ -0,0 +1,16 @@
+module 0x42::M {
+    // Incorrectly named constants
+    const Another_BadName: u64 = 42; // Should trigger a warning
+    const YetAnotherName: u64 = 777; // Should not trigger a warning
+
+    const minimumValue: u64 = 10; 
+    const some_important_number: u64 = 55;
+    
+    const HttpTimeout: u64 = 30000; 
+    const JSON_Max_Size: u64 = 1048576;
+    const VALUE_MAX: u64 = 200;
+
+    // Should not trigger a warning
+    #[allow(lint(constant_naming))]
+    const numItems: u64 = 30;   
```

Comment:
```suggestion
```
Let's remove this. Tests should test one thing, with the suppression, it is testing two: the errors and the suppression of the errors

Reply:
removed